### PR TITLE
Added FAQ Article Regarding Instrument To PSLab Documentation Website

### LINKED ISSUE #83: Questions Regarding how to use the Compass in PSLab as well as How it Works
**Describe the issue**
While most people have become fairly acquainted with the idea of everything becoming digital, few know how to use such applications and the inner working of such gadgets

**Would you like to work on the issue?**
Yes I would like to work on the issue.

### PR DESCRIPTION
**Pull request by aryanguls2**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [x] Feature implementation
- [x] Doc updates

* **Related Issue**
Fixes #83 
It helps users of PSLab get various questions answered on the website itself without having too search anywhere else or on the internet.

* **What changes have you introduced?**
I have added a FAQ article wicg contains relevant images, links and information regarding the Compass Instrument. It contains information regarding its usage, how it works etc.

@mariobehling @hpdang @cweitat Mentors please review as soon as possible.

### CODE REVIEW

**Comment by aryanguls** [comment]
@cweitat Could you also review and merge this request for the FAQ article.

**Comment by aryanguls2** [comment]
@cweitat Sir could you also please review and merge this request.

**Comment by NishithP2004** [comment]
@aryanguls2 I don't think that it's a good practice to create two a/c 's just to get some work done in less time.

File: tutorials/Compass_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <td><img src="/images/Compass3.jpg"></td>
+        <td><img src="/images/Compass2.jpg"></td>
+    </tr>
+</table>
+
+Frequently Asked Questions And Their Answers
+=============================================
+While many of us consider ourselves to know quite a lot about technology there is always something more to be learnt. Through this article I wish to clear all general and frequent questions about the functioning and uses of the PSLab Compass. Thus in case you want some of your doubts cleared and questions answered this is the right place.
+
+1) How does the digital compass ?
```
Comment (NishithP2004) [inline tutorials/Compass_FAQ.md]: How does... ?
Do what ?

**Comment by aryanguls2** [comment]
@NishithP2004 @cweitat I have made the changes to the file now. Please review and accept merge request.

**Comment by aryanguls2** [comment]
@NishithP2004 this second account was a private one that I don't use for gci and used to use it for practice. But I forgot to log out recently and so 2 or 3 things might be from this account. But I don't plan to use it any further. I noticed it too late.

**Comment by aryanguls2** [comment]
See the point of this FAQ article is not just to suffice for now but can also be updated as the compass in the app gets updated. Newer featured will need newer tutorials and they can be added to the compass.md file but newer questions can be added to this. The whole point of the FAQ article is so that people who don't understand using the tutorial can also effectively use the compass by along questions about topics not mentioned in the tutorial or of things they were unable to understand. @cweitat

**Comment by mariobehling** [comment]
Closing it therefore.

**Pull request closed without merging**